README.md

```diff
@@ -1,5 +1,26 @@
 google-map
 ==========
 
-See https://elements.polymer-project.org/elements/google-map
+[![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://beta.webcomponents.org/element/GoogleWebComponents/google-map)
 
+<!---
+```
+<custom-element-demo>
+  <template>
+    <script src="../webcomponentsjs/webcomponents-lite.js"></script>
```

Comment:
can this be the webcomponents-lite.min.js file or will that mess up the inline demo? We should encourage folks to use that.